launchable/commands/split_subset.py

```diff
@@ -81,8 +95,65 @@ def run(self):
                 payload = {
                     "sliceCount": count,
                     "sliceIndex": index,
+                    "sameBins": [],
                 }
 
+                if same_bin_files is not None:
+                    same_bins = []
+                    for same_bin_file in same_bin_files:
+                        with open(same_bin_file, "r") as f:
+                            """
+                            A same_bin_file expects to have a list of tests with one test per line.
+                            Each line of test gets formatted and packed to sameBins list in payload.
+                            E.g.
+                                For gradle:
+                                ```
+                                $ cat same_bin_file.txt
+                                example.AddTest
+                                example.DivTest
+                                example.SubTest
+                                ```
+                                Formatted:
+                                ```
+                                "sameBins" [
+                                    [
+                                        {"class": "example.AddTest"},
+                                        {"class": "example.DivTest"},
+                                        {"class": "example.SubTest"}
+                                    ]
+                                ]
+                                ```
+                            E.g.
+                                For gotest:
+                                ```
+                                $ cat same_bin_file.txt
+                                example.BenchmarkGreeting
+                                example.ExampleGreeting
+                                ```
+                                Formatted:
+                                ```
+                                "sameBins" [
+                                    [
+                                        {"class": "example", "testcase": "BenchmarkGreeting"},
+                                        {"class": "example", "testcase": "ExampleGreeting"}
+                                    ]
+                                ]
+                                ```
+                            """
+                            t = f.readlines()
+                            t = [s.strip() for s in t]
+                            d = [self.same_bin_formatter(s) for s in t]
+                            same_bins.append(d)
+                    for i, s in enumerate(same_bins):
+                        for j, t in enumerate(same_bins):
+                            if i != j:
+                                for u in s:
+                                    if u in t:
+                                        raise ValueError(
```

Comment:
How about showing a warning message instead of a error not to stop the command?

e.g)
```
example.Add, exampleSub were included more than once so put together
```

Reply:
Fixed.
https://github.com/launchableinc/cli/pull/470/commits/4cc4762cbaae4e10b2a35b8320a3e7b7037ae520